src/util.cc

```diff
@@ -622,3 +624,46 @@ bool Truncate(const string& path, size_t size, string* err) {
   }
   return true;
 }
+
+bool ReplaceContent(const string& file_dst, const string& new_content,
+                    string* err) {
+#ifdef _WIN32
+  if (!DeleteFile(file_dst.c_str())) {
```

Comment:
I would still like to know *why* unlink fails on the CI. I'm having a similar problem in #1562.

Reply:
I sadly dont know, however, we have a codepath for windows and linux anyways, so i guess this does not really make a difference ...